CHANGELOG.md

```diff
@@ -2,10 +2,17 @@
 
 ## Unreleased
 
+### Bugfixes
+
+- **Onboarding Service Provider**
+  - add missing fields to form legal name & company identifier[#1151](https://github.com/eclipse-tractusx/portal-frontend/pull/1151)
+
 ## 2.3.0-RC2
 
 ### Bugfixes
 
+- **App Subscription Management**
+  - fixed 'read more' link by adding link [#1200](https://github.com/eclipse-tractusx/portal-frontend/pull/1200)
```

Comment:
Why are you adding this changelog entry?

Reply:
Now removed.